packages/block-library/src/nextpage/editor.native.scss

```diff
@@ -0,0 +1,9 @@
+// @format
+
+.block-library-nextpage__container {
+	align-items: center;
+	padding-left: 4;
```

Comment:
Does RN version of sass support shortcuts present in CSS like `padding: 4px`?

@hypest this is confusing to have something which looks like CSS but isn't. Maybe we should revisit this approach and start using JS files with style objects instead?

Reply:
padding: 4px worked! 👍

Reply:
> Does RN version of sass support shortcuts present in CSS like padding: 4px

I think it does, one just needs to provide all 4 or more of the individual fields. Example: `padding: 4px 4px 4px 4px`.

> @hypest this is confusing to have something which looks like CSS but isn't. Maybe we should revisit this approach and start using JS files with style objects instead?

I think we haven't really put any new effort on the styling side of things so, let's defer this convo for a bit. I think we will do style related work later in the year.